docs/installing/Installation-using-makefile.rst

```diff
@@ -0,0 +1,37 @@
+.. Copyright 2021 tsuru authors. All rights reserved.
+   Use of this source code is governed by a BSD-style
+   license that can be found in the LICENSE file.
+
+:title: Installing Tsuru using the makefile recipe
+
+.. _installing_tsuru_local:
+
+Installing Tsuru using the makefile recipe
+========================================================
+
+This post will show how to install Tsuru using the makefile recipe.
+
+Prequisites
+---------------------
+* minikube and kubectl. if you don't have it yet, you can install it `here <https://minikube.sigs.k8s.io/docs/start/>`_, with minikube and `kubectl <https://kubernetes.io/docs/tasks/tools/>`_ properly installed, you are good to go.
```

Comment:
I don't think kubectl is required although it could be recommended in case of any troubleshooting needs...

Reply:
I get it now, I will work on it.